collections/_infographics/climate/map-temperature-change.md

```diff
@@ -3,14 +3,29 @@ layout:        infographic
 title:         "Map of temperature changes (1961–2019)"
 slug:          "map-temperature-change"
 redirect_from: "/map-temperature-change"
-published:     2022-07-26
+published:     2022-11-20
 weight:        99
 tags-scopes:   [ world ]
 tags-topics:   [ climate ]
-caption:       "The global temperature rise since 1961 is approximately 1.1°C. However, the warming rate is not the same for all locations: while the temperature in most of the planet's oceans has not increased by more than 0.8°C, most of the world's land is warming considerably faster. The highest warming rate on Earth (over 4°C in the last 60 years) is in the Arctic Ocean."
+caption:       "The global temperature rise since 1961 is approximately 1.1 °C. However, the warming rate is not the same for all locations: while the temperature in most of the planet's oceans has not increased by more than 0.8 °C, most of the world's land is warming considerably faster. The highest warming rate on Earth (over 4 °C in the last 60 years) is in the Arctic Ocean."
 data-our:      "https://docs.google.com/spreadsheets/d/16bTWzt0y8Omne9xxjd3o1rpszF764ATaC5UpFO5Zd7I/edit?usp=sharing"
 data-orig:
   - [ "NASA GISS", "https://data.giss.nasa.gov/gistemp/maps/index_v4.html" ]
 ---
 
-{% include includes-local/comment-placeholder.md slug="infografiky/mapa-zmeny-teploty" %}
+## Additional comments and links
+
+* The world has warmed by about 1 °C since 1961<!--(see infographics [Global temperature anomaly](/infografiky/teplotni-anomalie))-->. However, the landmasses are warming at about twice the rate of the oceans, with the northern hemisphere warming faster than the southern hemisphere. Furthermore, warming varies at different times of the year - for example, warming in the Arctic Ocean during the winter months has exceeded +6 °C since 1961.
+
+* The amplification of warming over land and in the Northern Hemisphere is due to feedback in the climate system and is well modelled by current climate models. Back in 1979, the Charney Report summarized the results of climate models at the time, and the authors concluded, "...if the concentration of CO<sub>2</sub> in the atmosphere is doubled and thermodynamic equilibrium is reached, there will be a warming of 2-3.5 °C, with higher increases at northern latitudes."
```

Comment:
This is more an editorial comment than a linguistic one, but I would vote for quoting the Charney report directly here (option 2 of Willem's suggestions). At least in anglophone contexts, the quotation marks suggest no paraphrasing - a direct quote. So it would read:

Back in 1979, the Charney Report summarized the results of climate models at the time, and the authors concluded: 
"When it is assumed that the CO, content of the atmosphere is doubled and statistical thermal equilibrium is achieved, the more realistic of the modeling efforts predict a global surface warming of between 2°C and 3.5°C, with greater increases at high latitudes."

If we wanted to paraphrase, I can come up with something, but I think this way is better.

Reply:
Ok, I changed it to quoting the [Charney report](http://www.columbia.edu/~jeh1/Documents/charney_report1979.pdf).

Note that I fixed the `CO,` (incorrectly copied and pasted here) to use the proper subscript: `CO<sub>2</sub>`.

Reply:
In general, I would vote more for paraphrasing than direct quote. Reasoning: The direct quote is more complicated to understand (uses more sophisticated language). To me, the more important point is to get the message across (the original source is linked anyway for those interested).

But it's not crucial, I believe, we can keep it as is now.